MIGRATING-0.6-0.7.md

```diff
@@ -331,10 +367,28 @@ It now also disallows out of bounds drawing by default. This behaviour can be ch
 
 ## Style module
 
-The `style` module has been removed.
+The `style` module has been removed. The items in it have been moved:
 
-TODO: Improve this section before release.
+- `PrimitiveStyle` and `PrimitiveStyleBuilder` are now available under `embedded_graphics::primitives::{PrimitiveStyle, PrimitiveStyleBuilder}`.
+- `TextStyle` and `TextStyleBuilder` are now available under `embedded_graphics::text::{TextStyle, TextStyleBuilder}`.
+
+  Note that usage with `Text` has changed. See [the text changes section](#Text-rendering) for more.
 
 - `PrimitiveStyle` & `PrimitiveStyleBuilder` -> `primitives` module
 - `TextStyle` & `TextStyleBuilder` -> `text` module + see text changes
-- `Styled` -> crate root
+- `Styled` is now exported from the crate root, e.g. `use embedded_graphics::Styled`.
```

Comment:
`Styled` is now only used for primitives and is included in the `primitives` module.